### ISSUE TITLE
mdbx_sync_locked: Assertion `!(((head)->mm_datasync_sign) > 1u) || env->me_sync_pending != 0' failed.

### ISSUE THREAD

**Issue by dreamsxin** [leo-yuriev/libmdbx#69]
ver `0.3.1`
Use `MDBX_CREATE | MDBX_DUPFIXED` happen
```out
mdbx:5338: mdbx_sync_locked: Assertion `!(((head)->mm_datasync_sign) > 1u) || env->me_sync_pending != 0' failed.
```
Source:
```c

	rc = mdbx_env_create(&intern->env);
	if (rc != MDBX_SUCCESS) {
		printf("%s\n", mdbx_strerror(rc));
		return;
	}

	mdbx_env_set_maxreaders(intern->env, 1024);
	mdbx_env_set_mapsize(intern->env, 1024 * 1024 * 1024);
	mdbx_env_set_maxdbs(intern->env, 256);

	rc = mdbx_env_open(intern->env, "lmdbfrontend/", 0, 0664);
	if (rc != MDBX_SUCCESS) {
		printf("%s\n", mdbx_strerror(rc));
		return;
	}

	rc = mdbx_txn_begin(intern->env, NULL, 0, &intern->txn);
	if (rc != MDBX_SUCCESS) {
		printf("%s\n", mdbx_strerror(rc));
		return;
	}

	rc = mdbx_dbi_open(intern->txn, NULL, MDBX_CREATE | MDBX_DUPSORT | MDBX_DUPFIXED, &intern->dbi);
	if (rc != MDBX_SUCCESS) {
		printf("%s\n", mdbx_strerror(rc));
		return;
	}
	rc = mdbx_txn_commit(intern->txn);
```

**Comment by leo-yuriev** [comment]
This assertion is obsolete, i.e. the bug is that I didn't remove it. I will fix this in `stable/0.2` and `devel` branches soon.

You need to either wait for these fixes in a stable form in the `master` branch, or simply remove this `assert()` in your copy.

**Comment by dreamsxin** [comment]
Thank you.
